### PR TITLE
feat: added modal component to CL

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**
image
image

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
Looks good to me

File: CSS/base/reset.css [hunk 1]
```diff
@@ -108,8 +109,8 @@ button {
     text-align: center;
     margin: 1rem 0rem;
     font-weight: bold;
-    border: 1px solid var(--dark-primary-text);
-    box-shadow: 1px 1px 2px 2px var(--dark-primary-text);
+    /* border: 1px solid var(--dark-primary-text);
```
Comment (tusharkandpal) [inline CSS/base/reset.css:112]: Delete this commented code, please.

**Review by tusharkandpal — APPROVED** [review]
Clean code! Good to merge.

**Comment by ajit-sharmaGH** [comment]
> Looks good to me

Thank you!

**Comment by ajit-sharmaGH** [comment]
> Clean code! Good to merge.

Thank you !

**Pull request merged**